src/main/java/app/controller/impl/GameControllerImpl.java

```diff
@@ -123,4 +123,14 @@ public void delete(Context ctx) {
             throw new NotFoundResponse(e.getMessage());
         }
     }
+
+    public void getAllReviews(Context ctx) {
+        try {
+            Long id = ctx.pathParamAsClass("id", Long.class).get();
+
+            ctx.redirect(String.format("/api/reviews?id=%d", id));
```

Comment:
Should the query parameter be changed to gameId instead of id (ie. `/api/reviews?gameId=%d`) so it makes more sense in context and won't be confused for a review id after the redirect?